sql/migrate/migrate.go

```diff
@@ -191,6 +195,43 @@ func WithStateReader(dsr StateReader) PlannerOption {
 	}
 }
 
+// DisableHashSum disables the hash-sum functionality for the migration directory.
+func DisableHashSum() PlannerOption {
```

Comment:
Maybe `DisableSumFile`?

```suggestion
func DisableSumFile() PlannerOption {
```